tc/core/polyhedral/memory_promotion.h

```diff
@@ -111,9 +111,10 @@ class TensorReferenceGroup {
   TensorReferenceGroup() {}
 
  public:
-  static TensorGroups accessedBySubtree(
-      const detail::ScheduleTree* tree,
-      const Scop& scop);
+  static TensorGroups accessedWithin(
```

Comment:
Isn't the TC convention to document the declaration rather than the implementation?
(I don't mind, personally; I find it more convenient to have the documentation close to the implementation.)